llvm/test/MC/AMDGPU/custom-uc-version.s

```diff
@@ -0,0 +1,41 @@
+// RUN: llvm-mc -triple=amdgcn -show-encoding -mcpu=gfx1010 %s | FileCheck --check-prefix=GFX10 %s
+
+// Test that UC_VERSION* symbols can be redefined.
```

Comment:
This sounds like we want them be redefinable for some reason whereas it's actually quite the opposite.

As it is, this seem to simply test that symbols can be redefined, for which we probably already have some generic tests in MC/AsmParser?

Reply:
Just state this is nonsense and we just don't want to assert

Reply:
This test would pass with or without the code change.
If we think it is useful to check then I can add it back.